code/game/area/almayer.dm

```diff
@@ -98,23 +98,23 @@
 	resin_construction_allowed = FALSE
 
 /area/almayer/command/securestorage
-	name = "\improper Secure Storage"
+	name = "\improper Lower Deck Secure Storage"
 	icon_state = "corporatespace"
 	fake_zlevel = 2 // lowerdeck
 
 /area/almayer/command/computerlab
-	name = "\improper Computer Lab"
+	name = "\improper Lower Deck Computer Lab"
 	icon_state = "ceroom"
 	fake_zlevel = 2 // lowerdeck
```

Comment:
Aren't these two on upper deck?

Reply:
I'm not sure, I don't think I've ever gone to either haha, I was just going off the fake zlevel thing. I will put those back anyways, thanks.